tweak: blind with another effect

Honker9999999 opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Теперь заклинание ослепления ослепляет всех людей (кроме заклинателя) в радиусе 7 тайлов от заклинателя на 10 секунд.
Плюсом удалил механику, что если "время действия гена от магии больше кулдауна, ген не будет снят". Я хз зачем эта механика вообще была, учитывая что единственное что она делает - это бесконечное превращение в халка если максимально улучшить заклинание Mutate.
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/1219014729742159952/1225697986848227410
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

Honker9999999 committed:
blind with another effect

Gottfrei commented on code/datums/spells/genetic.dm:
```diff
@@ -24,8 +24,7 @@
 
-		if(duration < base_cooldown)
-			addtimer(CALLBACK(src, PROC_REF(remove), target), duration, TIMER_OVERRIDE|TIMER_UNIQUE)
+		addtimer(CALLBACK(src, PROC_REF(remove), target), duration, TIMER_OVERRIDE|TIMER_UNIQUE)
```
верни как было, это оптимизон, чтобы не создавать лишние таймеры, когда эффект спелла гарантированно завершится ранее, чем можно будет кастовать следующий.

Honker9999999 committed:
Update genetic.dm

Honker9999999 committed:
Update wizard.dm

Изменяет эффект от блайнда ещё немного (не критично. Изначально эти эффекты длились 10 секунд и 20 соотвественно, но потом мы перешли на децисекунды - но вот про эти забыли чето (по словам автора переноса))

Honker9999999 committed:
Update wizard.dm

Gottfrei commented on code/datums/spells/wizard.dm:
```diff
@@ -327,19 +327,18 @@
 /obj/effect/proc_holder/spell/inflict_handler/blind
-	amt_eye_blind = 10
-	amt_eye_blurry = 20
+	amt_eye_blind = 10 SECONDS
+	amt_eye_blurry = 20 SECONDS
```
мыло на 20с. в ТЗ нет.

Honker9999999 committed:
Update wizard.dm
